interop-testing/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
             project(':grpc-testing'),
             libraries.junit,
             libraries.mockito,
+            libraries.netty_tcnative,
```

Comment:
Just for starters, I do know of other tests that are using OpenSSL: the interop-testing unit tests are run internally at google using tcnative (I just confirmed). Tcnative is also used for the actual interop testing, both internally (I just confirmed) and [externally](https://grpc-testing.appspot.com/job/gRPC_interop_master/2200/HTML_Report/) (granted, those are a different sort of test).

I think the original plan was to use tcnative on Travis (because otherwise, why do a99e9779?), but still allow small-time contributors to use Jetty ALPN. I know we hit a snag with OkHttp, because it only supports Jetty ALPN on OpenJDK and it couldn't talk to Netty on Linux because OpenSSL is generally too old to include ALPN. That's a different problem than on OS X and Windows, because on those platforms OpenSSL is generally not available and the code falls back to Jetty ALPN.

So for Linux we needed to check whether OpenSSL is new enough. But that [was broken](https://github.com/netty/netty-tcnative/issues/58). I think once it was fixed we simply failed to follow through. Time to follow through; I've sent out #1332.